airflow/api/common/proto/variable.proto

```diff
@@ -0,0 +1,14 @@
+syntax = "proto2";
+
+package airflow;
+
+message Variable {
```

Comment:
Nothing is added to the variable table. This is the message scheme description. Protobufs are used to have a schema and to be really fast.